app/controllers/admin/users_controller.rb

```diff
@@ -1,8 +1,9 @@
 class Admin::UsersController < ApplicationController
   before_action :error_unless_admin
   def show
+    @user = User.find(params[:id])
   end
   def index
-    @users = User.where(role: 0)
+    @user = User.where(role: 0)
```

Comment:
Why did we rename this to @user singular if it is supposed to be many users?

Reply:
Switched it back, good catch